src/GraphQL/Execution/ExecutionError.cs

```diff
@@ -1,17 +1,21 @@
 using System;
+using GraphQL.Language;
 
 namespace GraphQL
 {
     public class ExecutionError : Exception
     {
-        public ExecutionError(string message)
-            : base(message)
+        public Field Field { get; set; }
+
+        public ExecutionError(Field field, string message)
+            : this(field, message, null)
         {
         }
 
-        public ExecutionError(string message, Exception innerException)
+        public ExecutionError(Field field, string message, Exception innerException)
```

Comment:
I wouldn't want to include the whole field object here.  Also since it doesn't always apply, I wouldn't want to make it a required field in the constructor.  I would probably be fine with adding a `string` of `FieldName` as a optional field.